src/ec/packages/ec-component-footer-harmonised/footer-harmonised.story.js

```diff
@@ -1,52 +1,412 @@
-/* eslint-disable no-param-reassign */
+/* eslint-disable no-param-reassign, no-shadow */
 import { storiesOf } from '@storybook/html';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { withKnobs, button, text } from '@storybook/addon-knobs';
 import withCode from '@ecl-twig/storybook-addon-code';
+
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import logoEC from '@ecl/ec-resources-logo/logo--en.svg';
-
 import dataGroup1 from './demo/data--group1';
 import dataGroup2 from './demo/data--group2';
 import dataGroup3 from './demo/data--group3';
 
 import footerHarmonised from './ecl-footer-harmonised.html.twig';
 import notes from './README.md';
+// Preserve the original data.
+const logo0 = JSON.parse(JSON.stringify(dataGroup3.sections[0].logos[0]));
+const logo1 = JSON.parse(JSON.stringify(dataGroup3.sections[0].logos[1]));
+let dataG1 = [...dataGroup1.sections];
+const dataG2 = [...dataGroup2.sections];
+const dataG3 = [...dataGroup3.sections];
+// Labels for the buttons.
+const requiredGroupId = 'Mandatory elements';
+const optionalGroupId = 'Optional elements';
+const useCasesGroup = 'Use cases';
+
+// Buttons callbacks for optional elements.
+// Group 3
+const partnershipLogoBtnToggler = () => {
+  dataG3[0].logos[0] = dataG3[0].logos[0] ? false : logo0;
```

Comment:
Whitout partnershipLogo we still have `<li>` with empty `<img>` tags
```
<ul class="ecl-footer-harmonised__logo-list">
  <li class="ecl-footer-harmonised__logo-item"><img src="" alt="" title="" class="ecl-footer-harmonised__logo" /></li>
  <li class="ecl-footer-harmonised__logo-item"><img src="" alt="" title="" class="ecl-footer-harmonised__logo" /></li>
  <li class="ecl-footer-harmonised__logo-item"><img src="static/media/logo--en.30b933cc.svg" alt="European Commission logo" title="European Commission" class="ecl-footer-harmonised__logo" /></li>
</ul>
```
Isn't it better to really delete logo objects?

Reply:
It is. :)